src/items.cpp

```diff
@@ -1058,9 +1047,9 @@ SDL_Surface* itemSprite(Item* item)
 			node_t* node = list_Node(&items[item->type].surfaces, spell->ID);
 			if ( !node )
 			{
-				return NULL;
+				return nullptr;
 			}
-			SDL_Surface** surface = (SDL_Surface**)node->element;
+			auto** surface = static_cast<SDL_Surface**>(node->element);
```

Comment:
unnecessary auto, recommend leave as SDL_Surface**

Reply:
Fixed in 3010621c